src/ucp/wireup/select.c

```diff
@@ -1314,6 +1341,8 @@ ucp_wireup_select_wireup_msg_lane(ucp_worker_h worker,
 
     ucp_wireup_fill_aux_criteria(&criteria, ep_params);
     for (lane = 0; lane < num_lanes; ++lane) {
+        ucs_assert_always((lane_descs[lane].rsc_index != UCP_NULL_RESOURCE) &&
```

Comment:
why needed?

Reply:
wireup_msg_lane is not actual until CM server doesn't have fallback logic, will be useful in the future